docs/gallery/static/impulse_response_measurement.ipynb

```diff
@@ -0,0 +1,500 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "ax = pf.plot.freq(1/x_reference)\n",
+    "ax.set_ylim(-80, 0)\n",
+    "ax.set_title('Direct inversion of the excitation signal')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The solution to this problem is to applied a regularized inversion that limits the inversion effort outside a pre-defined bandwidth. This can be realized with the [`pyfar.dsp`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.html) module"
```

Comment:
```suggestion
    "The solution to this problem is to apply a regularized inversion that limits the inversion effort outside a pre-defined bandwidth. This can be realized with the [`pyfar.dsp`](https://pyfar.readthedocs.io/en/stable/modules/pyfar.dsp.html) module"
```

Reply:
Link is displayed at https://pyfar-gallery.readthedocs.io/en/enhancement-measurement_nb_static/gallery/static/impulse_response_measurement.html